GitUp/Application/WelcomeWindow.m

```diff
@@ -150,38 +245,11 @@ - (void)willShowPopUpMenu {
   }
 }
 
-#pragma mark - Setup
-- (void)setupButtons {
-  // buttons
-  self.twitterButton.textAlignment = NSLeftTextAlignment;
-  self.twitterButton.textFont = [NSFont boldSystemFontOfSize:11];
-  self.forumsButton.textAlignment = NSLeftTextAlignment;
-  self.forumsButton.textFont = [NSFont boldSystemFontOfSize:11];
-  [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(willShowPopUpMenu) name:NSPopUpButtonWillPopUpNotification object:self.recentPopUpButton];
-}
-- (void)setup {
-  self.opaque = NO;
-  self.backgroundColor = [NSColor clearColor];
-  self.movableByWindowBackground = YES;
-  [self setupButtons];
-}
-
-- (void)awakeFromNib {
-  [self setup];
-}
-
-#pragma mark - Actions
-- (BOOL)validateMenuItem:(NSMenuItem*)menuItem {
-  return menuItem.action == @selector(performClose:) ? YES : [super validateMenuItem:menuItem];
-}
-
-- (void)performClose:(id)sender {
-  [self.appDelegate closeWelcomeWindow:sender];
-}
-
-#pragma mark - Window
-- (BOOL)canBecomeKeyWindow {
-  return YES;
+#pragma mark - Actions/Twitter&Issues
+- (void)openTwitter {
+  if (self.model.twitterURL) {
```

Comment:
This model URL isn't needed. Welcome window is the only one using the Twitter URL.